api/src/main/java/com/capitalone/dashboard/config/LoggingFilter.java

```diff
@@ -108,6 +108,13 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             }
 
         } else {
+            //new code added 
+			//adds headers to response to allow CORS	
+			httpServletResponse.addHeader("Access-Control-Allow-Origin", "*");
```

Comment:
@tabladrum @satishc1 @amitmawkin  - should we be allowing any origin to access Hygieia.By using * in Access-Control-Allow-Origin, we enabled every application to  send a XHR request to Hygieia and I think we should have a control/logging of calling application by creating a collection of whitelisted urls and enabling them in "Access-Control-Allow-Origin" by replacing * with listOfUrls. 

Eg :  "Access-Control-Allow-Origin":"*"
Replaced with : "Access-Control-Allow-Origin":  listOfUrls 

where listOfUrls - "http://google.com,http://dummy.com,....etc"

That way we have access to enable to particular resources instead of all.